README.md

```diff
@@ -7,116 +7,119 @@ visibility into the runtime state of the system. The framework is intended
+    python3 vol.py —f /home/user/samples/stuxnet.vmem windows.info
     ```
-4. Run some other plugins. The -f or —-single-location is not strictly
+
+4. Run some other plugins. The `-f` or `—-single-location` is not strictly
 required, but most plugins expect a single sample. Some also
-require/accept other options.  Run "python vol.py <plugin> -h"
+require/accept other options.  Run `python3 vol.py <plugin> -h`
 for more information on a particular command.
 
 ## Symbol Tables
 
-Symbol table packs for the various operating systems are available for download at: <br>
+Symbol table packs for the various operating systems are available for download at:
 
-https://downloads.volatilityfoundation.org/volatility3/symbols/windows.zip <br>
-https://downloads.volatilityfoundation.org/volatility3/symbols/mac.zip <br>
-https://downloads.volatilityfoundation.org/volatility3/symbols/linux.zip <br>
+<https://downloads.volatilityfoundation.org/volatility3/symbols/windows.zip>  
+<https://downloads.volatilityfoundation.org/volatility3/symbols/mac.zip>  
+<https://downloads.volatilityfoundation.org/volatility3/symbols/linux.zip>  
 
-Symbol tables zip files must be placed, as named, into the volatility/symbols <br>
-directory (or just the symbols directory next to the executable file). <br>
-Window symbols that cannot be found will be queried, downloaded, generated and <br>
-cached.  Mac and Linux symbol tables must be manually produced by a tool such <br>
-as dwarf2json. <br>
+Symbol tables zip files must be placed, as named, into the `volatility/symbols` directory (or just the symbols directory next to the executable file).
 
-Please note: These are representative and are complete up to the point of <br>
-creation for windows and mac.  Due to the ease of compiling linux kernels and <br>
-the inability to uniquely distinguish them, an exhaustive set of linux symbol <br>
-tables cannot easily be supplied. <br>
+Windows symbols that cannot be found will be queried, downloaded, generated and cached.  Mac and Linux symbol tables must be manually produced by a tool such as [dwarf2json](https://github.com/volatilityfoundation/dwarf2json).
+
+Please note: These are representative and are complete up to the point of creation for Windows and Mac.  Due to the ease of compiling Linux kernels and the inability to uniquely distinguish them, an exhaustive set of Linux symbol tables cannot easily be supplied.
 
 ## Documentation
 
-The framework is documented through doc strings and can be built using sphinx. <br>
+The framework is documented through doc strings and can be built using sphinx.
 
-The latest generated copy of the documentation can be found at: <br>
-https://volatility3.readthedocs.io/en/latest/ <br>
+The latest generated copy of the documentation can be found at: <https://volatility3.readthedocs.io/en/latest/>
 
 ## Licensing and Copyright
 
-Copyright (C) 2007-2019 Volatility Foundation
+Copyright © 2007-2019 Volatility Foundation
```

Comment:
In case people do read this in a text editor, I'd feel happier if this were `(C)` rather than `©`.

Reply:
Yeah, that's fair enough. I'll swap it back.

Reply:
Done